deploy/kubernetes/base/daemonset.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: apps/v1beta2 # for versions before 1.8.0 use extensions/v1beta1 [apps/v1beta2]
+kind: DaemonSet
+metadata:
+  name: cadvisor
+spec:
+  template:
+    spec:
+      containers:
+      - name: cadvisor
+        image: gcr.io/google-containers/cadvisor:v0.30.2
```

Comment:
We want to make sure this is kept up-to-date, but I'm not sure what the best way to do that is... We could have a test that parses the release notes headers and ensures this is kept in sync?

Reply:
Ideally, we want it to track whatever we consider the "latest version" version to be, right?  So maybe add it to the "updating latest" part of the release?